addons/website_event_track/static/src/scss/event_track_templates.scss

```diff
@@ -150,6 +178,32 @@
     }
 }
 
+@if $o-wevent-migration-layer {
+    .o_we_online_agenda_overlay {
+        @include o-position-absolute(0, 0, 63px);
+        background-image: linear-gradient(90deg, rgba($body-color, 0), rgba($body-color, 0.1), rgba($body-color, 0.3));
+        opacity: 0;
+        transition: opacity .1s ease;
+    }
+
+    .o_we_online_agenda_has_scroll {
+        border-top-color: $border-color;
+
+        + .o_we_online_agenda_overlay {
+            opacity: 1;
+            transition: opacity .5s ease;
+        }
+    }
+
+    .o_we_online_agenda_has_scroll.o_we_online_agenda_is_scrolling + .o_we_online_agenda_overlay {
+        opacity: .5;
+    }
+
+    .o_we_track_day_header {
+        z-index: 2;
+    }
+}
+
```

Comment:
Ah I get it now, I thought the overlay was going to be more visible, that's why I asked if it was done. It works quite nicely actually